transport_nantes/mailing_list/templates/mailing_list/mailing_list_list_view.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'asso_tn/base_mobilitain.html' %}
+
+{% block content %}
+<div class="container-fluid p-3">
+	<h2 class="text-center">Listes (newsletters, etc.)</h2>
+	<table class="table table-bordered table-striped table-responsive-sm">
+		<thead class="thead-dark">
+		<tr>
+			<th scope="col">Nom</th>
+			<th scope="col">Fréquence des contacts par semaine(s)</th>
+			<th scope="col">État</th>
+			<th scope="col">Nombre d'inscrits</th>
+		</tr>
+		</thead>
+		<tbody>
+		{% for mailing_list,count_subscriber in mailing_lists %}
+		<tr>
+			<td>{{ mailing_list.mailing_list_name }}</td>
+			<td>{{ mailing_list.contact_frequency_weeks }}</td>
+			<td>{% if mailing_list.list_active %} Activer {% else %} Désactiver {% endif %}</td>
```

Comment:
"Activé" if you're talking about its status, "Activer" if you mean to place a button to activate the list. It's an action verb.

Reply:
Isn't it "active" -- une liste active ou pas active ?